### PR TITLE
Initial Support for Laravel 5.3

### PR DESCRIPTION
**Pull request by luceos**
This is a WIP. Awaiting release of 5.3.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ install:
   - mysql -e "create database hyn;" -uroot
   - mysql -e "grant all privileges on *.* to 'travis'@'localhost' with grant option;" -uroot
   # create a new laravel project in the subfolder laravel (default composer behaviour)
-  - composer require laravel/laravel 5.2
+  - composer require laravel/laravel dev-develop
```
Comment (luceos) [inline .travis.yml:32]: We need to test against both 5.2 and 5.3.

File: composer.json [hunk 2]
```diff
@@ -16,10 +16,11 @@
     },
     "require": {
         "php": ">=7.0.0",
-        "laravel/framework": "~5.2.0",
+        "laravel/framework": "~5.3.0",
```
Comment (luceos) [inline composer.json:19]: We'll not hardcode 5.3, because then we won't support 5.2

File: .travis.yml (hunk 1 above)
Comment (bkintanar) [inline .travis.yml:32]: Ok. I'll work on this later.

**Comment by luceos** [comment]
PR was a bit premature.

**Pull request closed without merging**